src/foam/comics/v2/DAOCreateView.js

```diff
@@ -82,7 +86,13 @@ foam.CLASS({
     {
       name: 'save',
       code: function() {
-        this.config.dao.put(this.data).then((o) => {
+        var clonedData = this.data.clone();
```

Comment:
Shouldn't the clone be inside the if statement, since that's the only case it is actually necessary? So you do:
var cdata = this.data; 
if (LcAware.isInstance(cdata)) { 
  cdata = cdata.clone(); 
  ...

Reply:
sounds good will account for it so theres no unnecessary clones